.github/workflows/release.yml

```diff
@@ -0,0 +1,172 @@
+name: release
```

Comment:
Let's rename this to `controller-release.yaml` and `name: controller-release`.

Reply:
Fixed, thanks!